### ISSUE TITLE
Add support for GPG symmetric key

### ISSUE THREAD

**Issue by Ogglas** [magnumripper/JohnTheRipper#1251]
Got this error when trying to run gpg2john. File available below:

https://www.dropbox.com/s/kvigykny4p5ech5/Flag07.txt.gpg?dl=0

**Comment by magnumripper** [comment]
@kholia do you have some spare time?

**Comment by kholia** [comment]
@Ogglas do you know the corresponding password? if not, do you have a sample file, which is very similar?

**Comment by kholia** [comment]
I now recommend using PGPCrack-NG for cracking symmetrically encrypted PGP files (commit 195afcd62a64d29feecd550e94113f9fbbf48ab3).

https://github.com/kholia/PGPCrack-NG

**Comment by Ogglas** [comment]
Yes I managed to crack it, password is "Robert Fischer". PGPCrack-NG does not give you much feedback so it would be nice if John had support for it. If you need more files I have 12x with passwords.

**Comment by kholia** [comment]
@Ogglas 

Someone will need to write the patches to add the support. I am relieved to know that PGPCrack-NG was able to crack the password (without fucking up) ;)

**Comment by Ogglas** [comment]
Sorry Kholia but PGPCrack-NG dit not give me any feedback so I ended up writing a shell script that iterated a wordlist. :)

**Comment by kholia** [comment]
```
$ cat wordlist.txt | ./PGPCrack-NG Flag07.txt.gpg
Found Password : Robert Fischer
```

**Comment by Ogglas** [comment]
Wierd, I will try it again when I have time.

**Comment by magnumripper** [comment]
Fixed in 3996744ee
